src/Compilers/CSharp/Portable/Symbols/Source/SourcePropertySymbolBase.cs

```diff
@@ -164,6 +182,28 @@ private enum Flags : byte
             }
         }
 
+        internal SynthesizedBackingFieldSymbol CreateBackingField(bool isCreatedForFieldKeyword)
+        {
+            Debug.Assert(!IsIndexer);
+            if (_lazyBackingFieldSymbol is null)
+            {
+                var backingField = new SynthesizedBackingFieldSymbol(this,
+                                              GeneratedNames.MakeBackingFieldName(_name),
+                                              isReadOnly: (HasGetAccessor && !HasSetAccessor) || IsInitOnly,
+                                              this.IsStatic,
+                                              hasInitializer: (_propertyFlags & Flags.HasInitializer) != 0,
+                                              isCreatedForfieldKeyword: isCreatedForFieldKeyword);
+                InterlockedOperations.Initialize(ref _lazyBackingFieldSymbol, backingField);
+            }
+
+            if (isCreatedForFieldKeyword) // If not created for field keyword, the flag is set in SourcePropertySymbolBase ctor.
+            {
+                _propertyFlags |= Flags.IsAutoProperty; // PROTOTYPE(semi-auto-props): Revise precisely when we need to set this.
```

Comment:
>_propertyFlags [](http://example.com/codeflow?start=16&length=14)

We probably don't want to mutate the flags if the field was created due to a speculation through SemanticModel